programmers/난이도별/level02.올바른_괄호/dkdlelk99.py

```diff
@@ -0,0 +1,19 @@
+# 출처 https://programmers.co.kr/learn/courses/30/lessons/12909
+# input '(' 또는 ')'으로만 이루어진 문자열 s
+# output 올바른 괄호인지 아닌지에 대한 bool 값
+
+def solution(s):
+    a =''
+    if s.count('(') == s.count(')'):
+        for i in s:
+            a += i
+            if a[-2:] == '()':
+                a = a[:-2]
+        return True if a == '' else False
+    else:
+         return False
```

Comment:
```suggestion
    for i in s:
        a += i
        if a[-2:] == '()':
            a = a[:-2]
    return True if a == '' else False
```

사실 앞뒤로 묶은 if문도 필요 없습니다.

효율성 테스트 1에서 조금 더 효율이 떨어질 뿐 문제 없이 통과 가능